Tasks/CondaEnvironmentV0/task.json

```diff
@@ -49,23 +51,32 @@
             "defaultValue": true,
             "helpMarkDown": "Update Conda to the latest version. This applies to the Conda installation found in `PATH` or at the path specified by the `CONDA` environment variable."
         },
+        {
+            "name": "installOptions",
```

Comment:
Consider 'installArguments' and 'createArguments' to match the labels more closely.

Reply:
I would argue that "options" is the correct term here.  I would say that everything that comes after `conda create` is an "argument," including options, the list of package specs, and the name of the environment.

The point about the label is fair, though, so I will change it to say "additional options."  (The commands are already run with the `--quiet` and `--yes` options.)